### PR TITLE
[Task#68272]Add I18n to Project

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#68272](https://edu-redmine.sun-asterisk.vn/issues/68572)

## WHAT this PR does?
Add I18n to Project

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.
Screen Shot 2023-09-14 at 10 20 37

Screen Shot 2023-09-14 at 10 24 18

### CODE REVIEW

**Review by Hungnd-0370 — APPROVED** [review]

**Comment by tungbd-0214** [comment]
ready

File: resources/js/Context/LangContext.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+        <LangContext.Provider value={{ lang, changeLocale }}>
+            {children}
+        </LangContext.Provider>
+    );
+};
+
+export const useLang = () => {
+    return useContext(LangContext);
+};
```
Comment (ghost) [inline resources/js/Context/LangContext.jsx]: eof

File: resources/js/Layouts/Authenticated.jsx [hunk 2]
```diff
@@ -1,123 +1,159 @@
...
+                                    </Link>
+                                </div>
 
-                            <div className="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
-                                <NavLink href={route('dashboard')} active={route().current('dashboard')}>
-                                    Dashboard
-                                </NavLink>
+                                <div className="hidden space-x-8 sm:-my-px sm:ml-10 sm:flex">
+                                    <NavLink href={route('dashboard')} active={route().current('dashboard')}>
+                                        Dashboard
```
Comment (ghost) [inline resources/js/Layouts/Authenticated.jsx]: localization

File: resources/js/Layouts/Authenticated.jsx [hunk 3]
```diff
@@ -1,123 +1,159 @@
...
 
-                                    <Dropdown.Content>
-                                        <Dropdown.Link href={route('logout')} method="post" as="button">
-                                            Log Out
-                                        </Dropdown.Link>
-                                    </Dropdown.Content>
-                                </Dropdown>
+                                        <Dropdown.Content>
+                                            <Dropdown.Link href={route('logout')} method="post" as="button">
+                                                Log Out
```
Comment (ghost) [inline resources/js/Layouts/Authenticated.jsx]: localization

File: resources/js/Layouts/Authenticated.jsx [hunk 4]
```diff
@@ -1,123 +1,159 @@
...
-                    </div>
-
-                    <div className="pt-4 pb-1 border-t border-gray-200">
-                        <div className="px-4">
-                            <div className="font-medium text-base text-gray-800">{auth.user.name}</div>
-                            <div className="font-medium text-sm text-gray-500">{auth.user.email}</div>
+                    <div className={(showingNavigationDropdown ? 'block' : 'hidden') + ' sm:hidden'}>
+                        <div className="pt-2 pb-3 space-y-1">
+                            <ResponsiveNavLink href={route('dashboard')} active={route().current('dashboard')}>
+                                Dashboard
```
Comment (ghost) [inline resources/js/Layouts/Authenticated.jsx]: localization

File: resources/js/Layouts/Authenticated.jsx [hunk 5]
```diff
@@ -1,123 +1,159 @@
...
-                            </ResponsiveNavLink>
+                        <div className="pt-4 pb-1 border-t border-gray-200">
+                            <div className="px-4">
+                                <div className="font-medium text-base text-gray-800">{auth.user.name}</div>
+                                <div className="font-medium text-sm text-gray-500">{auth.user.email}</div>
+                            </div>
+
+                            <div className="mt-3 space-y-1">
+                                <ResponsiveNavLink method="post" href={route('logout')} as="button">
+                                    Log Out
```
Comment (ghost) [inline resources/js/Layouts/Authenticated.jsx]: localization

**Review by ghost — COMMENTED** [review]
ignore các file khởi tạo bởi hệ thống.

**Comment by tungbd-0214** [comment]
ready

**Comment by tungbd-0214** [comment]
ready

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**